keps/prod-readiness/sig-network/3866.yaml

```diff
@@ -0,0 +1,6 @@
+# The KEP must have an approver from the
+# "prod-readiness-approvers" group 
+# of http://git.k8s.io/enhancements/OWNERS_ALIASES
+kep-number: 3866
+alpha:
+  approver: ""
```

Comment:
Feel free to put me, I will add my comments after @soltysh is ok with it.

[Reading the KEP now]